osgp/platform/osgp-adapter-ws-smartmetering/src/main/resources/db/migration/V20210702135657832__Adds_table_application_key_configuration.sql

```diff
@@ -0,0 +1,21 @@
+DO
+$$
+BEGIN
+
+IF NOT EXISTS (
+    SELECT 1
+    FROM   pg_tables
+    WHERE  schemaname = current_schema
+    AND    tablename  = 'application_key_configuration') THEN
+
+    CREATE TABLE application_key_configuration (
+        organisation_identification VARCHAR(40) NOT NULL,
+        application_name VARCHAR(40) NOT NULL,
+        public_key_location VARCHAR(2000),
+        PRIMARY KEY (organisation_identification, application_name)
+    );
+
```

Comment:
Some time ago action was taken to make sure all tables and columns had comments on them explaining their purpose.
I think the new table and columns had better be accompanied by such comments as well.

For an example you could have a look at: "V20180817140448941__Adds_table_notification_web_service_configuration.sql".

Reply:
Added comments